### DISCUSSION TITLE
/trapped command

### DISCUSSION THREAD

**Discussion by sirlinium** [GriefPrevention/GriefPrevention#303] [Q&A] [answered]
Hello, please excuse me for posting wrong place...

would it be possible to be able to disable the /trapped use ?

many thanks !

**Comment by RoboMWM** [comment, marked as answer]
`griefprevention.trapped`

It's documented on the commands page, but not the permissions page, so if someone wants to update the permissions page with that that'd be appreciated.

**Comment by sirlinium** [comment]
I updated the page, thank you !
